ddtrace/llmobs/_writer.py

```diff
@@ -61,6 +61,7 @@ class LLMObsEvaluationMetricEvent(TypedDict, total=False):
     categorical_value: str
     numerical_value: float
     score_value: float
+    ml_app: str
```

Comment:
do we want to add `timestamp_ms` here as well?

Reply:
Ah yes, thanks for catching